packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -165,29 +251,95 @@ export interface TreeView<TSchema extends ImplicitFieldSchema> extends IDisposab
 	 */
 	upgradeSchema(): void;
 
+	/**
+	 * Initialize the tree, setting the stored schema to match this view's schema and setting the tree content.
+	 *
+	 * Only valid to call when this view's {@link SchemaCompatibilityStatus.canInitialize} is true.
+	 *
+	 * Applications should typically call this function before attaching a `SharedTree`.
+	 * @param content - The content to initialize the tree with.
+	 */
+	initialize(content: InsertableTreeFieldFromImplicitField<TSchema>): void;
+
 	/**
 	 * Events for the tree.
 	 */
 	readonly events: Listenable<TreeViewEvents>;
 }
 
 /**
- * Information about how a view schema was incompatible.
+ * Information about a view schema's compatibility with the document's stored schema.
+ *
+ * See SharedTree's README for more information about choosing a compatibility policy.
  * @public
  */
-export interface SchemaIncompatible {
+export interface SchemaCompatibilityStatus {
 	/**
-	 * True iff the view schema supports all possible documents permitted by the stored schema.
+	 * Whether the view schema allows exactly the same set of documents as the stored schema.
+	 *
+	 * @remarks
+	 * Equivalence here is defined in terms of allowed documents because there are some degenerate cases where schemas are not
+	 * equal in a strict sense but still allow the same documents, and the document notion is more useful to applications.
+	 *
+	 * Examples which are expressible where this may occur include:
+	 * - schema repository `A` has extra schema which schema `B` doesn't have, but they are unused (i.e. not reachable from the root schema)
+	 * - field in schema `A` has allowed field members which the corresponding field in schema `B` does not have, but those types are not constructible (ex: an object node type containing a required field with no allowed types)
+	 *
+	 * These cases are typically not interesting to applications.
+	 */
+	readonly isExactMatch: boolean;
+
+	/**
+	 * Whether the current view schema is sufficiently compatible with the stored schema to allow viewing tree data.
+	 * If false, {@link TreeView.root} will throw upon access.
+	 *
+	 * A necessary condition for this to be true is that the documents allowed by the view schema are a subset of those allowed by the stored schema.
+	 * This is not sufficient: the simple-tree layer does not tolerate read APIs which return out-of-schema data.
+	 * For example, if the view schema for a node has a required `Point` field but the stored schema has an optional `Point` field,
+	 * read APIs on the view schema do not work correctly when the document has a node with a missing `Point` field.
+	 * Similar issues happen when the view schema has a field with less allowed types than the stored schema and the document actually leverages those types.
```

Comment:
We should not have public docs explaining how APIs fail to work in cases that are impossible to cause.

Also simple-tree is an implementation detail, and should not be mentioned in public facing docs.

These could be privateRemarks docmments.

Public facing stuff should be more along the lines of:
"Trees can only be read using a view schema that can handle all possible documents comparable with the tree's stored schema. For example, and value listed as required in the view schema must be required in the stored schema. View schema also requires that all data is included in the schema, so all fields which can contain data in the stored schema must also be present, and compatible, in the view schema. If the view schema permits any cases not allowed by the stored schema, this would result in the editing APIs being able to produce invalid content (with respect to the stored schema) so even that case is considered incompatible and unable to view. In the future APIs may be added to opt into various possibles for how to handle these incompatibilities (like allowing ignoring unexpected data, or using defaults or adapters to convert and provide required data), allowing canView to be true in more cases."